src/app/static/src/js/utils/types.ts

```diff
@@ -2,3 +2,23 @@ export interface Parameter {
     name: string,
     value: string
 }
+
+export interface Instances {
+    source: string,
+    annexe: string
+}
+
+export type Dict<V> =  {[index: number]: V}
+
+export interface ReportLog{
+    email: string,
+    date: String,
+    report: string,
+    instances: Dict<Instances>,
+    params: Dict<Parameter>,
```

Comment:
This is stretching my TypeScript knowledge, but `instances` and `params` are both `Map<String, String>` in Kotlin so should be `Record<string, string>` here?

Also `git_branch` etc are optional, so should be `git_branch?: string`?